server/controllers/editEvent.js

```diff
@@ -0,0 +1,10 @@
+const eventEdit = require('../database/queries/eventEdit')
+
+const editEvent = (req, res) => {
+  const data = req.body
```

Comment:
you should validate the input! Which fields are required? (I'd say at least event_title, event_author and event_id). Does the input have expected format? (text, number, email...)

There are libraries to help with this, for instance https://www.npmjs.com/package/node-input-validator